packages/documentation/.storybook/preview.ts

```diff
@@ -81,6 +83,10 @@ const preview: Preview = {
             title: 'View full screen',
             onClick: openFullScreenDemo,
           },
+          {
+            title: 'Copy variation',
+            onClick: copyStoryConfigUrl,
```

Comment:
Is there the possibility to shortly display a different title to get feedback on the interaction?

-> Initially: "Copy link"
-> On successfully copying a link: "Copied!"
-> After 3 seconds: "Copy link" again